tf2_ros/src/tf2_ros/transform_broadcaster.py

```diff
@@ -39,9 +39,16 @@ class TransformBroadcaster:
     """
     :class:`TransformBroadcaster` is a convenient way to send transformation updates on the ``"/tf"`` message topic.
     """
+    def __init__(self, node, qos=QoSProfile(depth=100)):
```

Comment:
Same question about whether to support `node` creation here with an optional `node` argument...

Reply:
The [C++ `TransformBroadcaster` does not create a node](https://github.com/ros2/geometry2/blob/a59b75417e3a6e74ce5f9269e5c282877e3de0d2/tf2_ros/include/tf2_ros/transform_broadcaster.h#L51-L59). 

Even without that, I'd recommend against creating a node here since a hidden node comes with a bunch of problems including [unique node name issues](https://github.com/ros2/geometry2/issues/124), needing to get the hidden node into an executor (Applies to `TransformListener`, not `TransformBroadcaster`), and I think being unable to remap `/tf` and `/tf_static` on the hidden node when dynamically composing nodes.